src/Sesion15.java

```diff
@@ -0,0 +1,33 @@
+import java.util.Scanner;
+
+public class Sesion15 {
+
+    public static void main(String[] args){
+        Sesion15 sesion15=new Sesion15();
+        sesion15.switchCase();
+
+    }
+
+    public void switchCase(){
+
```

Comment:
@gaur673 please remove extra space from the file.

Reply:
removed